client/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+How to develop on this python client project 
+=
+<hr />
+
+Armada Project welcomes contributions from the opensource community.
+### Note that you need PYTHON 3+ to get this project working and this project does not work on native windows OS.
+
+## Setting up your own fork of this repository on linux, Mac, and WSL OS.
+<hr />
+
+1) On github interface click on the `Fork` button.
+2) Clone your forked of armada repository on your personal git repository by running git clone `github.com:Your_Git_Username/armada.git`.
+3) Enter the directory `cd armada`
+4) Add upstream repository [Armada Repo](https://github.com/G-Research/armada.git)
+
+## Install Pyenv/Python and Poetry
```

Comment:
We're about to merge a change in how the client is built, which invalidates some of this thru no fault of your own.

```suggestion
## Install Pyenv/Python
```